bundles/org.openhab.binding.velux/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -160,19 +178,30 @@
 		<label>@text/channel-type.velux.state.label</label>
 		<description>@text/channel-type.velux.state.description</description>
 		<category>WallSwitch</category>
+		<tags>
+			<tag>Switch</tag>
```

Comment:
See previous comment. This wallswitch may do anything, from turining on heating, lighting etc. I don;t think these tags should go here.